RawSpeed/LJpegDecompressor.cpp

```diff
@@ -463,27 +429,25 @@ void LJpegDecompressor::createHuffmanTable(HuffmanTable *htbl) {
  ************************************/
 
 void LJpegDecompressor::createBigTable(HuffmanTable *htbl) {
-  const uint32 bits = 14;      // HuffDecode functions must be changed, if this is modified.
+  const uint32 bits = HuffmanTable::TableBitDepth;      // HuffDecode functions must be changed, if this is modified.
   const uint32 size = 1 << bits;
-  int rv = 0;
-  int temp;
-  uint32 l;
 
   if (!htbl->bigTable)
     htbl->bigTable = (int*)_aligned_malloc(size * sizeof(int), 16);
```

Comment:
> 1. it has nothing to do with the code changes in this PR (I did not touch this line)
> 2. it has nothing to do with the subject of the respective commit

Indeed, it was not a  review comment even though github misleadingly shows it as such.

> 3. the order of the operands in a int multiplication of two numbers is irrelevant. It produces the same exact assembler code: https://godbolt.org/g/5YCvtk
> 4. this particular multiplication could never overflow anyway, because both operands are compile time constants

Yes, i was not talking about *this particular* multiplication. As practice shows, code tends to get copied around, and if some piece of code does not follow the same pattern as the other similar pieces, it is likely to eventually end up being used in a code where it matters, and overflow there.

Though i'm mostly speaking about pointer overflows that we were having in darktable. There was issues with overflow/etc, which were fixed by casting to `size_t` first/placing `size_t`-sized variable first/etc.